Solutions/Marain.Workflow.Storage.Sql.Database/Marain.Workflow.Storage.Sql.Database.refactorlog

```diff
@@ -28,4 +28,11 @@
     <Property Name="ParentElementType" Value="SqlTable" />
     <Property Name="NewName" Value="SerializedWorkflow" />
   </Operation>
+  <Operation Name="Rename Refactor" Key="c461901b-fc70-40c5-9bb4-c297304c659d" ChangeDateTime="02/25/2020 11:45:35">
```

Comment:
Bearing in mind that this is nominally adding SQL support for the first time, could and should we be starting with a clean refactor log?

Reply:
Good question. Yes, I think we should - for simplicity down the line.